crates/cubecl-core/src/runtime_tests/atomic.rs

```diff
@@ -0,0 +1,193 @@
+use crate::{self as cubecl, ir::Elem, AtomicFeature, Feature};
+
+use cubecl::prelude::*;
+
+#[cube(launch)]
+pub fn kernel_atomic_add<I: Numeric>(output: &mut Array<Atomic<I>>) {
+    if UNIT_POS == 0 {
+        Atomic::add(&output[0], I::from_int(5));
+    }
+}
+
+fn supports_feature<R: Runtime, F: Numeric>(
+    client: &ComputeClient<R::Server, R::Channel>,
+    float_feat: AtomicFeature,
+) -> bool {
+    match F::as_elem_native_unchecked() {
+        Elem::Float(kind) => {
+            client
+                .properties()
+                .feature_enabled(Feature::FloatAtomic(float_feat))
+                && client
+                    .properties()
+                    .feature_enabled(Feature::Type(Elem::AtomicFloat(kind)))
```

Comment:
Feature::FloatAtomic => Feature::AtomicFloat to have the same pattern as the Elem::AtomicFloat. (Prefix instead of postfix)

Reply:
guess that also makes sense. I just figured the feature is floating point atomics, hence `FloatAtomics`. But consistency is also good.